tds/routers/associations.py

```diff
@@ -0,0 +1,94 @@
+"""
+router.associations - crud operations for associations and related tables in the DB
```

Comment:
On second thought, I think you're right about orchestration for this case because we might want to manage user associations with objects using orchestration